integration_tests/test_basic.py

```diff
@@ -250,5 +252,33 @@ def test_delete_participant(apps, as_admin):
 
     assert cell.count() == 0
 
+def test_merge_participant(apps, as_admin):
+
+    """ Test if two participant merge succesfully """
+
+    Experiment = apps.backend.get_model('experiments', 'Experiment')
+    Location = apps.backend.get_model('experiments', 'Location')
+    apps.backend.load('leader.json')
+    location = Location.objects.create(name="Test Lab")
+    page = as_admin
+
+    page.goto(f"{apps.backend.url}/participants/merge/")
+
+    page.click("#select2-id_old_participant-container")
+    old_option = page.locator("li.select2-results__option", has_text="[2] name unknown")
+    old_option.click()
+
+    page.click("#select2-id_new_participant-container")
+    new_option = page.locator("li.select2-results__option", has_text="[3] Han S. Olo")
+    new_option.click()
+
+    page.click("button.btn.btn-primary:has-text('Merge participants')")
+
+    success_alert = page.locator("div.alert.alert-success")
+    expect(success_alert).to_have_text("Participant merged!")
+    assert success_alert.is_visible()
 
-    
\ No newline at end of file
+    page.goto(f"{apps.backend.url}/participants/")
+    table_body = page.locator("#DataTables_Table_0 tbody")
+    rows_count = table_body.locator("tr").count()
+    assert rows_count == 1, f"Expected 1 participant, but found {rows_count}"
```

Comment:
This is good, but you could maybe add a further check that the merge worked as expected (e.g. the name is correct, email is correct etc. I'm not sure how it's supposed to behave). You could also do it directly with the participant model instead of reading it from the page (because at that point we are interested in checking the data layer)